regexes/device/mobiles.yml

```diff
@@ -16532,16 +16623,22 @@ Xiaomi:
     - regex: 'Redmi Note 5A Prime'
       model: 'Redmi Note 5A Prime'
       device: 'phablet'
-    - regex: 'M2003J15SC(?:[);/ ]|$)'
+    - regex: 'M2010J19SC(?:[);/ ]|$)'
       model: 'Redmi Note 9'
       device: 'phablet'
+    - regex: 'M2003J15SC(?:[);/ ]|$)'
```

Comment:
Where did you get the information from that `M2003J15SC` is a Note 10x and not a Note 9?

Reply:
this is a model for the China market, it is called there `Note 10x`
for the global market, it's called the `Note 9`

* http://storage.googleapis.com/play_public/supported_devices.html  (google result Note 10x)
* https://launchstudio.bluetooth.com/Listings/Search   (here it gives out as a note 9)
* https://www.gsmarena.com/xiaomi_redmi_10x_4g-10202.php
* https://xiaomifirmwareupdater.com/archive/firmware/merlin/ 

 more old models (good repository)
* https://github.com/KHwang9883/MobileModels

Reply:
Actually I don't have a preference here. Guess we can also use 10x then